encrypted-config-value/src/main/java/com/palantir/config/crypto/Base64Utils.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2015 Palantir Technologies, Inc. All rights reserved.
+ */
+
+package com.palantir.config.crypto;
+
+import java.util.Base64;
+
+public final class Base64Utils {
+
+    private Base64Utils() {
+        /* do not instantiate */
+    }
+
+    public static boolean isBase64(String value) {
+        try {
+            Base64.getDecoder().decode(value);
```

Comment:
see other comment (happy to chat in person) but there are subtleties around what to do with trailing lower-order bits